vocab/security/template.html

```diff
@@ -259,114 +255,110 @@ <h2>Diagram description</h2>
         <p>
           The diagram uses boxes, ellipses, and connecting lines with different "styles" 
           (border color, end marker, line type) to differentiate their semantic meaning; 
```

Comment:
This one-character change goes along with the larger change just below.
```suggestion
          (border color, end marker, line type) to differentiate their semantic meaning:
```